OWNERS

```diff
@@ -27,6 +23,5 @@ reviewers:
   - paveldournov
   - qimingj
```

Comment:
We can remove @qimingj and @rileyjbauer I think; I believe they have moved onto other projects.